sandbox/sandbox/apps/home/models.py

```diff
@@ -1,5 +1,9 @@
-from django.db import models
-from wagtail.core.models import Page
+from wagtail import VERSION as WAGTAIL_VERSION
+
+if WAGTAIL_VERSION >= (3, 0):
```

Comment:
**Suggestion**: I'm not sure the sandbox app needs to have older version compatibility added. 

The tests don't seem to use it.

Reply:
Reverted that particular change.